autoscheduler/scraper/models/section.py

```diff
@@ -12,29 +14,38 @@ class Section(models.Model):
     course_num = models.IntegerField(db_index=True)
     section_num = models.IntegerField(db_index=True)
     term_code = models.IntegerField(db_index=True)
-
     min_credits = models.IntegerField() # Will never be null
     max_credits = models.IntegerField(null=True) # Will be null in most cases
-
     max_enrollment = models.IntegerField()
     current_enrollment = models.IntegerField()
     instructor = models.ForeignKey('Instructor', on_delete=models.CASCADE)
 
     class Meta:
         db_table = "sections"
 
+def generate_meeting_id(section_num: str, meetings_count: str):
+    """ Generates the meeting id in the form of {section_num}{meetings_count}"""
+
+    return "".join((section_num, meetings_count))
+
+def generate_meeting_time(string_time: str):
+    """ Generates a meeting time from a string in format hhmm. ex) 1245 = 12:45am. 1830 = 6:30 pm."""
+    hour = int(string_time[0:2])
+    minute = int(string_time[2:4])
+    time = datetime.time(hour,minute)
+
+    return time
+

```

Comment:
While technically meetings are the only time we parse the meeting, I don't necessarily think it's necessary to have it in this file (rather, it should be in `scrape_courses.py`). Likewise, this needs to account for when the meeting times are empty, so you need to check if string_time is empty, and if it is, return `None`.